src/components/position-at-box-shape-border.js

```diff
@@ -122,7 +125,22 @@ AFRAME.registerComponent("position-at-box-shape-border", {
       const scale = this.halfExtents[inverseHalfExtents[targetHalfExtentStr]] * distance;
       const targetScale = Math.min(2.0, Math.max(0.5, scale * tempParentWorldScale.x));
 
-      this.target.scale.setScalar(targetScale / tempParentWorldScale.x);
+      if (!this.prevScale && Math.abs(this.prevScale - targetScale) > 0.01) {
```

Comment:
This logic breaks if you spawn a duck from a spawner while you're in pause mode. It also breaks the dynamic scaling behavior that currently exists on non-spawner media objects.